static/test/javascripts/conf/adblock.js

```diff
@@ -0,0 +1,9 @@
+module.exports = function (config) {
```

Comment:
Why do we need this configuration file? We are already pulling in specs in /test/javascripts/spec/common/*